dev/system-test/firestore.ts

```diff
@@ -173,6 +173,20 @@ describe('Firestore class', () => {
       unsubscribe();
     }
   });
+
+  it('throws an error if terminate() is called with pending BulkWriter operations', async () => {
+    const writer = firestore._bulkWriter();
+    const ref = randomCol.doc('doc-1');
+    writer.set(ref, {foo: 'bar'});
+    try {
```

Comment:
I still see the try/catch code here. Did you update this test to use `eventually.be.rejectedWith` as well?

Reply:
Confused which test you were referring to, but it has been now :)